upgrade_test.py

```diff
@@ -198,12 +202,40 @@ def rollback_node(self, node):
 
         self.upgradesstables_if_command_available(node)
 
-    def upgradesstables_if_command_available(self, node):
+    def upgradesstables_if_command_available(self, node, queue=None):
         upgradesstables_supported = node.remoter.run(
             'nodetool help | grep -q upgradesstables && echo "yes" || echo "no"')
         if "yes" in upgradesstables_supported.stdout:
             self.log.debug("calling upgradesstables")
             node.remoter.run('nodetool upgradesstables -a')
+        if queue:
+            queue.put(node)
+            queue.task_done()
+
+    def wait_for_sstable_upgrade(self, node, queue=None):
+
+        def wait_for_node_to_finish():
+            result = node.remoter.run('sudo find /var/lib/scylla/data/system -type f | xargs -I{} basename {}')
+            all_sstable_files = result.stdout.splitlines()
+
+            sstable_version_regex = re.compile(r'(\w+)-\d+-(.+)\.(db|txt|sha1|crc32)')
+
+            sstable_versions = list(
+                set([sstable_version_regex.search(f).group(1) for f in all_sstable_files if sstable_version_regex.search(f)]))
+
+            assert len(sstable_versions) == 1, "expected all table format to be the same found {}".format(sstable_versions)
+            assert sstable_versions[0] == 'mc', "expected to format version to be '{}', found '{}'".format(
+                self.expected_sstable_format_version, sstable_versions[0])
+
+        try:
+            self.debug("Starting to wait for upgardesstables to finish")
```

Comment:
If upgradesstables command isn't available, we don't need to wait.

Reply:
you got a point here :)
I'll fix it

Reply:
fixed that